qemu/tests/qemu_guest_agent.py

```diff
@@ -2125,6 +2149,7 @@ def gagent_check_fsfreeze_list(self, test, params, env):
                 if self.gagent.get_fsfreeze_status() == \
                         self.gagent.FSFREEZE_STATUS_FROZEN:
                     self.gagent.fsthaw(check_status=False)
+        self.gagent_setsebool_value('off', params, self.vm)
```

Comment:
Move this one to `if params.get("os_type") == "linux":` block too.

Reply:
oh,left it ,sorry,edited yet.